manifests/service.pp

```diff
@@ -11,6 +11,7 @@
   file { $service_file_location:
     content => template($service_file_template),
     mode    => '0755',
+    before  => Service[confluence],
```

Comment:
One recommendation would be to manage it the way @hunner is doing it:

https://github.com/hunner/puppet-hiera/blob/master/manifests/init.pp#L109-L112